CONTRIBUTING.md

```diff
@@ -168,6 +170,29 @@ Please remember to:
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels ((enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any ['help wanted'](https://github.com/aws/aws-step-functions-data-science-sdk-python/labels/help%20wanted) issues is a great place to start.
 
 
+## Setting Up Your Development Environment
+
+Setting up your IDE for debugging your tests locally will save you a lot of time.
+You might be able to `Run` and `Debug` the tests directly in your IDE with your default settings, but if it's not the case,
+follow the steps described in this section.
+
+### PyCharm
```

Comment:
add a sub-heading to indicate that this is specific to running unit tests, unless this is talking about actual IDE setup

Reply:
+1
since this is specific to running tests, will add a sub heading as suggested